### PR TITLE
Add the missing radius mix/max and scale control for GeoJSON layer

### PR DESCRIPTION
**Pull request by shaojingli**
@Pessimistress as we discussed yesterday.

### CODE REVIEW

**Comment by shaojingli** [comment]
@Pessimistress should we call the new props `pointRadius***` or `markerRadius***` or something else?

**Comment by Pessimistress** [comment]
We don't use the word "marker" anywhere. I like "point" because it refers to the corresponding GeoJson feature type.

File: docs/layers/geojson-layer.md [hunk 1]
```diff
@@ -112,6 +112,23 @@ Type of joint. If `true`, draw round joints. Otherwise draw miter joints.
 The maximum extent of a joint in ratio to the stroke width.
 Only works if `lineJointRounded` is `false`.
 
+##### `pointRadiusScale` (Number, optional)
+
+- Default: `1`
+
+Global radius across all points.
```
Comment (Pessimistress) [inline docs/layers/geojson-layer.md]: Global radius multiplier?

Comment (shaojingli) [inline docs/layers/geojson-layer.md]: I'm just copying from the scatterplot layer doc... I will change both

**Comment by shaojingli** [comment]
I feel the same. Was asking because we use "marker" in our scatterplot layer doc. I will clean up

**Pull request merged**